docs/utilities/batch.md

```diff
@@ -783,6 +786,7 @@ Inheritance is importance because we need to access message IDs and sequence num
 
 Use the context manager to access a list of all returned values from your `record_handler` function.
 
+##### - TODO - is this a quote or example? is the right place/ fit?
```

Comment:
@heitorlessa  tagging you to get your attention here when you will review the work

Reply:
Great catch - I had added to add more explanation then didn't remove it.